src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -729,4 +729,11 @@ object messages {
            | - auxiliary constructors specify the implicit value
            |"""
   }
+
+  case class NoClassOrObjectAfterCase()(implicit ctx: Context)
+  extends Message(27) {
+    val kind = "Syntax"
+    val msg = hl"Only ${"class"} or ${"object"} are allowed after ${"case"}."
+    val explanation = ""
```

Comment:
The only possible explanation I can think of is for `case trait`. Do you think there can be any more meaningful token used after case?